src/config/resolve-config-editorconfig.js

```diff
@@ -3,11 +3,23 @@
 const path = require("path");
 
 const editorconfig = require("editorconfig");
-const mem = require("mem");
+const memoize = require("fast-memoize");
 const editorConfigToPrettier = require("editorconfig-to-prettier");
 const findProjectRoot = require("./find-project-root");
 
-const jsonStringifyMem = (fn) => mem(fn, { cacheKey: JSON.stringify });
+const memoizeCache = new Map();
+const memoizeOptions = {
+  cache: {
+    create() {
+      return {
+        has: (key) => memoizeCache.has(key),
+        get: (key) => memoizeCache.get(key),
+        set: (key, value) => memoizeCache.set(key, value)
+      };
+    }
+  }
+};
+const jsonStringifyMem = (fn) => memoize(fn, memoizeOptions);
```

Comment:
I would be happy to help get the lint:typecheck error resolved here once we get my other comments and questions resolved.